app/src/main/java/com/douglasbruce/tasky/core/common/utils/DateUtils.kt

```diff
@@ -0,0 +1,20 @@
+package com.douglasbruce.tasky.core.common.utils
+
+import java.time.Instant
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.time.ZoneId
+
+sealed class DateUtils {
+
+    companion object {
+        fun getDateMilli(date: LocalDate): Long {
+            return date.atStartOfDay(ZoneId.of("UTC")).toInstant().toEpochMilli()
+        }
+
+        fun getLocalDate(dateMilli: Long): LocalDate {
+            return LocalDateTime.ofInstant(Instant.ofEpochMilli(dateMilli), ZoneId.of("UTC"))
+                .toLocalDate()
+        }
+    }
+}
```

Comment:
How comes this is a sealed class? If you have a class that only contains a companion object, it's equivalent to just having a normal `object`

Reply:
Originally had more functionality that was removed. Just never revisited the idea of no longer needing a sealed class